serverless/installing_serverless/upgrading-serverless.adoc

```diff
@@ -8,6 +8,39 @@ toc::[]
 
 If you have previously installed a Technology Preview version of {ServerlessProductName}, follow the instructions in this guide to upgrade to the latest version.
 
+[id="update-api-group_{context}"]
+== Updating the API group
+
+In {ServerlessProductName} 1.4.0 and newer versions, the `KnativeServing` object's API group changed from `serving.knative.dev` to `operator.knative.dev`.
+You must adjust any of your scripts or applications that rely on the old API group to use the new API group.
+
+The {ServerlessOperatorName} creates a `KnativeServing` custom resource definition (CRD) in the new API group for you.
```

Comment:
@abrennan89 Sorry for confusion, Ashleigh. But I really meant changing just this single sentence to use CRD. It doesn't make so much sense in the other parts of documentation. We have CRD and CR (both of them), it's just the serverless operator will automatically create only the CRD. And the user has to manually create the CR.

Reply:
The user mostly manipulates with `CR`. The only part where they have to work with CRD is when removing it via `oc delete crd knativeservings.serving.knative.dev`

Reply:
No worries, sorry it just seems a bit confusing that we go between the two. Is this accurate now?